api/app/models/source_file_change.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Context on this comment: https://blog.saeloun.com/2024/05/20/frozen-string-literal/

TL;DR it freezes string literals, making them immutable. I added this so that the constants for the change category cannot be modified. 

Normally this comment should be added to all files, but for some reason I didn't do it 🤷